docker_mcp/services/stack.py

```
@@ -489,3 +724,102 @@
+            
+        Returns:
+            List of expected mount strings in format "source:destination"
+        """
+        try:
+            import yaml
+            compose_data = yaml.safe_load(compose_content)
+            expected_mounts = []
+            
+            # Parse services for volume mounts
+            services = compose_data.get("services", {})
+            for service_name, service_config in services.items():
```

Ruff (0.12.2)

Loop control variable `service_name` not used within loop body

Rename unused `service_name` to `_service_name`

(B007)